R/system.R

```diff
@@ -1089,17 +1089,41 @@ select_columns <- function(x, ...) {
   return (df)
 }
 
+clear_cach_file <- function(url){
+  hash <- digest::digest(url, "md5", serialize = FALSE)
+  tryCatch({
+    do.call(rm, c(as.name(hash)),envir = .GlobalEnv)
```

Comment:
I think it is better to call unlink() to remove the file. The tempfile page says that is the way to remove file.